src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -216,8 +218,12 @@ private void ThumbResizeThumbOnDragDeltaHandler(object sender, DragDeltaEventArg
 
         private void CompositionTargetRenderingHandler(object sender, EventArgs e)
         {
-            ViewModel.UpdateNearClipPlane();
-            ViewModel.ComputeFrameUpdate();
+            if (!View.Camera.Position.Equals(prevCamera) && !View.IsMouseCaptured )
```

Comment:
what does this do internally is this a reference comparison or does Point3d override this?